terraform/deployments/govuk-publishing-platform/outputs.tf

```diff
@@ -17,6 +17,19 @@ output "signon_security_groups" {
   description = "Used by ECS RunTask to run short-lived tasks with the same SG permissions as the signon ECS Service."
 }
 
+output "content-store" {
+  value = {
+    draft = {
+      task_definition_cli_input_json = module.draft_content_store.cli_input_json,
+      security_groups                = module.draft_content_store.security_groups,
```

Comment:
based on the concourse pipeline, am I correct to say that there will be a task definition/service update if the security-group is only updated by not the cli_input_json? One would think only the `cli_input_json` warrants a new task definition/service update.

Reply:
Yep, you're quite right.

There are some downstream deploys (not for content store, yet, but for publisher) which do depend on the security group though - the run-task thing we're using for DB migrations needs to know the security groups (so the task has permission to talk to the DB).

At the moment, those tasks get deployed in the same job as the task definition, so we want them to trigger when the SGs change as well as when the task definition changes.

Reply:
Ummm... I'm not convinced about this because the security groups to the DBs are set once while the DB migrations happen when the developer introduces a new schema/migration.